src/main/java/seedu/dietmanager/DailyFoodRecord.java

```diff
@@ -84,13 +86,12 @@ public String showBreakfast() {
     public String showLunch() {
         String message = "";
         for (Food food : lunch) {
-            message = message + food.getFoodName() + ", ";
+            message = message + food.toString();
         }
         if (message.equals("")) {
-            message = "Oops, you have no record for this meal";
-            return message;
+            message = "Oops, you have no record for this meal.\n";
```

Comment:
magic string

Reply:
again, not written by me, but i can shift it to bank